app/src/main/java/com/jjswigut/matters/database/MatterDataBaseDao.kt

```diff
@@ -0,0 +1,31 @@
+
+
+package com.jjswigut.matters.database
+
+import androidx.lifecycle.LiveData
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.Query
+
+@Dao
+interface MatterDataBaseDao{
+
+    @Insert
+    suspend fun insert(matter: Matter)
+
+    @Insert
+    suspend fun update(matter: Matter)
+
+    @Query("SELECT * from matter_table WHERE matterId = :key")
+    suspend fun get(key: Long): Matter?
+
+    @Query("DELETE FROM matter_table")
+    suspend fun clear()
+
+    @Query("SELECT * FROM matter_table ORDER BY matterId DESC LIMIT 1 ")
+    suspend fun getMatter(): Matter?
```

Comment:
how is this one going to be used in the future? It would seem to only return 1 note?

Reply:
I was thinking about using this to retrieve the note that is clicked on when we navigate to the edit part?_

Reply:
I'd imagine we would need to order by ID in our SQL statement though. this is just going to always give us the first note in the DB which I don't think is our desired behavior